CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -10,6 +10,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### Package `clientv3`
 - Fix [runtime panic that occurs when KeepAlive is called with a Context implemented by an uncomparable type](https://github.com/etcd-io/etcd/pull/18937)
+- Compile binaries using [go 1.22.10](https://github.com/etcd-io/etcd/pull/19004).
```

Comment:
Please add this item under `## Dependencies`

Reply:
There is already a section `## Dependencies` below (line 20), please add this item there.

Reply:
below line 20 , is the section of etcd grpc-proxy, do you want me to add the same thing there ?

Reply:
Yes, the section is not nested. The dependencies listed there are not from `etcd grpc-proxy`. Thanks!